<launch>
   <!-- name of the hunt to execute -->
   <arg name="hunt_name" default=""/>
   <!-- RGB image topic for image tasks -->
   <arg name="image_topic" default=""/>
   <!-- topic for image detection: yolo, etc -->
   <arg name="image_detection_topic" default=""/>

   <!-- Service end points exposed by scavenger hunt API -->
   <arg name="get_hunt_srv" default="/scavenger_hunt/get_hunt"/>
   <arg name="send_proof_srv" default="/scavenger_hunt/send_proof"/>
   <arg name="get_proof_srv" default="/scavenger_hunt/get_proof_status"/>

   <node name="huntLauncher" type="huntLauncher" pkg="scavenger_interface">
	<param name="hunt_name" value="$(arg hunt_name)"/>
   </node>
</launch>
